inf-clojure.el

```diff
@@ -238,11 +238,17 @@ This should usually be a combination of `inf-clojure-prompt' and
 `inf-clojure-subprompt'."
   :type 'regexp)
 
-(defcustom inf-clojure-prompt-on-set-ns t
+(defcustom inf-clojure-prompt-on-commands t
```

Comment:
That's a bit vague. Maybe "prompt-for-symbol-at-point" or something. And you forgot to update the docstring.